### ISSUE TITLE
WebGL Build for Map Renderer

### ISSUE THREAD

**Issue by nikstar802** [microsoft/MapsSDK-Unity#21]
I tried to create a webgl build for the citytour example. But unfortunately it seems BingMaps information is not loading in webgl. I am seeing completely blank picture. 

Can anyone help in this case? Is this SDK supports webgl build??

**Comment by kircher1** [comment]
Unfortunately the SDK doesn't support WebGL and it isn't something we plan on supporting in the future. The WebGL platform in Unity doesn't allow for using multi-threaded async Tasks which is something the SDK needs to efficiently process and prepare data for rendering. This incompatibility is one of the large reasons that we are unable to support WebGL.

**Comment by RomArtefacto** [comment]
Hello, as I understood multi-threaded async Tasks is now be supported in the latest versions of unity, is there any new plan on your side to also support WebGL in a near future ?

**Comment by kircher1** [comment]
@romgithubb, could you share more details on the Unity support? My understanding is that async/await Tasks in C# still aren't compatible.

If that is unblocked though, there is at least one more hurdle. This SDK has dependencies on native libraries, which can't be used in WebGL.

**Comment by romgithubb** [comment]
I have seen this some months ago https://forum.unity.com/threads/multithreading-and-webgl.817986/ , some unity comments was promising but indeed it seems not so clear today. on my side i currently have no more info, i'll come back to you if i find any updates regarding that topic
